module5/projects/Capstone.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: page
+title: Capstone Project
+---
+
+Before you dive full force into the job-hunt, we want you to start work on a Capstone Project!  This project should highlight what you have learned during Mods 1 - 5, and will be a great way to differentiate your portfolio from your peers.
+
+This will be an individual project, and the product is largely up to you!  We will give you some guidelines to make sure you are demonstrating your learning, but the topic and much of the functionality of your project is all your own.
+
+The project will be split into 4 Phases:
+* Ideation
+* Inception
+* Development
+* Presentation/Documentation
+
+### Project Requirements
+Your application must include the following:
```

Comment:
This feels like a lot for two weeks, but I do think we want them all pushing for this.

I'm just trying to think if we want to have softer wording than "requirements". Maybe "Project Expectations". Just trying to think of what we do if a student project doesn't meet these "requirements" like they don't have full crud working.

Reply:
Good call - I like project expectations... I'll also include something about prioritizing the API consumption.